packages/webpack-cli/src/webpack-cli.ts

```diff
@@ -421,12 +423,12 @@ class WebpackCLI implements IWebpackCLI {
         let skipInstallation = false;
 
         // Allow to use `./path/to/webpack.js` outside `node_modules`
-        if (dependency === WEBPACK_PACKAGE && fs.existsSync(WEBPACK_PACKAGE)) {
+        if (dependency === WEBPACK_PACKAGE && WEBPACK_PACKAGE_IS_CUSTOM) {
```

Comment:
I think we should use `require.resolve()` here and improve error reporting here (same for below) if package was not found